src/payment/strategies/adyenv2/adyenv2-payment-strategy.ts

```diff
@@ -71,6 +72,16 @@ export default class AdyenV2PaymentStrategy implements PaymentStrategy {
 
                 this._adyenComponent = adyenComponent;
 
+                const adyenCustomCardComponent = this._adyenCheckout.create('securedfields', {
+                    onChange: (state: AdyenCardState) => {
+                        this._updatePaymentMethodState(state);
+                    },
+                });
+
+                adyenCustomCardComponent.mount(`#${paymentMethod.id}-tsv`);
```

Comment:
The container ID needs to be passed in instead of being hardcoded here. The user of the SDK needs to be able to specify where the component can be inserted.